### ISSUE TITLE
nothing silently seems to happen when run from the CLI

### ISSUE THREAD

**Issue by alexgill** [tmcw/gedcom#45]
<pre>
agill@adar81:~/Documents/family_tree/GRAMPS/temp$ which gedcom
/usr/local/bin/gedcom
agill@adar81:~/Documents/family_tree/GRAMPS/temp$ gedcom
agill@adar81:~/Documents/family_tree/GRAMPS/temp$ gedcom Untitled_1.ged output.json
agill@adar81:~/Documents/family_tree/GRAMPS/temp$ ls
Untitled_1.ged
agill@adar81:~/Documents/family_tree/GRAMPS/temp$ 
</pre>

The GEDCOM file to convert was created in Gramps.
I gather this isn't a support forum and I could be stepping through the code, but any help would help.  TIA.

**Comment by matthelliwell2** [comment]
@alexgill did you ever get this working? I've got the same issue.

**Comment by alexgill** [comment]
Hi!

Nope I didn't yet.

But I've been peaking at the code. The version installed with npm is a 
minified version, and the code up on Github is in Typescript as you 
might have noticed. I closed the issue b/c I intend to figure it out 
myself, although there's no telling when... I'd encourage you to do the 
same and if you do figure it out, I'd be happy to know.

Thanks,
Alex


On Sun, Aug 29, 2021 at 09:20:33AM -0700, matthelliwell2 wrote:
***@***.*** did you ever get this working? I've got the same issue.
